src/styles/blocks/contact-us.scss

```diff
@@ -17,6 +17,10 @@
     line-height: 140%;
     color: #3e3e3e;
     text-decoration: none;
+
+    &__hover {
+      cursor: pointer;
```

Comment:
It is said to change text color on hover:
![image](https://github.com/mate-academy/layout_miami/assets/46566190/6f492162-19a5-4386-8994-e8050dc1b917)

Reply:
I can not find such examples of colors in the mockup